lib/s3blaster.js

```diff
@@ -481,7 +481,7 @@ class S3Blaster {
             this.values = this.sizes.map(size => genObjs(size));
         }
         this.md5s = this.values.map(val =>
-            crypto.createHash('md5').update(val).digest('hex'));
+            crypto.createHash('md5').update(val, 'binary').digest('hex'));
```

Comment:
It is not necessary: https://nodejs.org/docs/v4.5.0/api/crypto.html#crypto_hash_update_data_input_encoding